tests/check.py

```diff
@@ -32,9 +32,9 @@ def passes(self, lst):
 
         errors = []
         for text in lst:
-            errors.append(self.this_check.check(text))
+            errors += self.this_check.check(text)
 
-        return len(errors[0]) == 0
+        return len(errors) == 0
```

Comment:
This isn't right — checks returns errors that are lists with the error code, location, message, etc. `append` adds each of these elements to the list, whereas we want the entire list added to the list. In this case, it makes no difference because we're only testing whether the list of errors is empty (and so your switch to `+=` doesn't break anything), but in general it's good to keep in mind that errors are returned as lists.

Reply:
The thing I foresee with this is that it can potentially call `append` multiple times depending on the number of checks that are passed. If it only checks the length of the first list it adds, what's to say the rest don't contain errors?

Reply:
Oops, you're right. Sorry about that. Please excuse me if it seems like I don't know anything about Proselint, it's been a while ;)

Can you open a new PR with just the changes on lines 35/37?

Looks like this must have always been broken. Maybe none of the tests ever send in a list of sentences vs. one at a time…

Reply:
That's okay, honestly I'm just grateful that you're attentive and sharing your thoughts on things. Sure, though, I'll open a new PR shortly.

Although, it does seem that no tests ever send in a list. Perhaps it would make testing faster to convert some of them, given that a loop inside of a function would be significantly faster than calling the function several times.